src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/FakeContentTypeRegistryService.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Test.Formatting
+{
+    internal class FakeContentTypeRegistryService : IContentTypeRegistryService
```

Comment:
One interesting implication here that's worth pointing out is that prior to this our language server testing didn't depend on VS concepts IIRC. I don't have a problem with us adding this because indirectly it does in fact depend on it but thought it was worth mentioning.

Reply:
> language server testing didn't depend on VS concepts IIRC

The language server testing depends on an HTML formatter, and the HTML formatter depends on VS services. If we don't want VS concepts in the language server testing, there is exactly one option: use a different language server for HTML services.